test/models/financialDetails/DocumentDetailSpec.scala

```diff
@@ -96,34 +96,31 @@ class DocumentDetailSpec extends UnitSpec {
     "originalAmountIsNotZeroOrNegative" should {
       "return false" when {
         "original amount is zero" in {
-          fullDocumentDetailModel.copy(originalAmount = Some(0)).originalAmountIsNotZeroOrNegative shouldBe false
+          fullDocumentDetailModel.copy(originalAmount = 0).originalAmountIsNotZeroOrNegative shouldBe false
         }
         "original amount is negative" in {
-          fullDocumentDetailModel.copy(originalAmount = Some(-20)).originalAmountIsNotZeroOrNegative shouldBe false
+          fullDocumentDetailModel.copy(originalAmount = -20).originalAmountIsNotZeroOrNegative shouldBe false
         }
       }
       "return true" when {
-        "original amount is not present" in {
-          fullDocumentDetailModel.copy(originalAmount = None).originalAmountIsNotZeroOrNegative shouldBe true
-        }
+//        "original amount is not present" in {
```

Comment:
I would assume this code is not applicable anymore and we can drop it? or this is something still need to be fixed?

Reply:
It can be dropped, I commented it out and forgot to delete it entirely from there after the discussion we had